service-bus.api.md

```diff
@@ -0,0 +1,256 @@
+## API Report File for "@azure/service-bus"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { AmqpMessage } from '@azure/core-amqp';
+import { DataTransformer } from '@azure/core-amqp';
+import { delay } from '@azure/core-amqp';
+import { Delivery } from 'rhea-promise';
+import Long from 'long';
+import { MessagingError } from '@azure/core-amqp';
+import { OperationOptions } from '@azure/core-auth';
+import { RetryOptions } from '@azure/core-amqp';
+import { TokenCredential } from '@azure/core-amqp';
+import { TokenType } from '@azure/core-amqp';
+import { WebSocketImpl } from 'rhea-promise';
+import { WebSocketOptions } from '@azure/core-amqp';
+
+// @public
+export interface CorrelationFilter {
+    contentType?: string;
+    correlationId?: string;
+    label?: string;
+    messageId?: string;
+    replyTo?: string;
+    replyToSessionId?: string;
+    sessionId?: string;
+    to?: string;
+    userProperties?: any;
+}
+
+// @public
+export interface CreateBatchOptions extends OperationOptions {
+    maxSizeInBytes?: number;
+    retryOptions?: RetryOptions;
+}
+
+export { DataTransformer }
+
+// @public
+export interface DeadLetterOptions {
+    deadLetterErrorDescription: string;
+    deadLetterReason: string;
+}
+
+export { delay }
+
+export { Delivery }
```

Comment:
`Delivery` was being exported before as it was part of the service bus message. Since we no longer expose this in the message, we don't need to export it